da_dk.lang

```diff
@@ -5,7 +5,6 @@ gui.xaero_delete=Slet
 gui.xaero_OK=OK
 gui.xaero_add_edit=Tilføj/Rediger
 gui.xaero_new_waypoint=Nyt Vejpunkt
-gui.xaero_initial=initial
```

Comment:
Oh yes, you are right. The word "initial" is used for initials in Danish as well, I don't know how I could have missed it. I'm going to readd it.

Reply:
Word "initial" is correct in Dutch, that's right, but in the source U.S. English translation it has been changed to "initials". As a result, it should be plural here as well, but since I don't know Danish, I decided to delete it and let the fallback mechanism do its job.

Reply:
Word "initial" is correct in Dutch, that's right, but in the source U.S. English translation file it has been changed to "initials". That's the reason why I decided to remove it and let the fallback mechanism do its job. I could have tried to translate it, but since I don't know Danish, I can never be 100% sure it is correct.

Reply:
Got it! Makes sense.